### PR TITLE
Ad Fast Fetch: move anchor click listener to adsense/doubleclick impl

### PR DESCRIPTION
**Pull request by keithwrightbos**
Move document click listener for FIE which expands CLICK_X/CLICK_Y into AdSense/Doubleclick impls and do not install if amp-ad-exit extension exists in creative (assuming it will handle expansion).

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -843,7 +854,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         // the size header for fluid ads. Fluid size headers always come back as
         // 0x0.
         !(size.width == 0 && size.height == 0);
-    setStyles(dev().assertElement(this.iframe), {
+    setStyles(dev().assert(this.iframe), {
```
Comment (glevitzky) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Why this change? `assertElement` calls `assert`.

Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Nevermind, just saw your recent commit.

Comment (keithwrightbos) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: I reverted... had an issue where I was attempting to make this.iframe as an object instead of element causing this to fail.
